pkg/cloudprovider/aws/apis/v1alpha1/register.go

```diff
@@ -32,6 +32,8 @@ var (
 	AWSRestrictedLabelDomains = []string{
 		"k8s.aws",
 	}
+	OperatingSystemBottleRocket = "Bottlerocket"
+	OperatingSystemEKSOptimized = "EKSOptimized"
```

Comment:
Sorry, should've mentioned this earlier but can we change this to EKSOptimizedLinux? There's also [EKS Optimized Windows AMIs](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-windows-ami.html) so we'd need to disambiguate.